src/main/java/forceitembattle/manager/ItemDifficultiesManager.java

```diff
@@ -44,6 +71,7 @@ public boolean itemInList(Material material) {
     }
 
     public ItemDifficultiesManager() {
+        this.descriptionItems = new ArrayList<>();
         this.easy = Arrays.asList(

```

Comment:
I just merged another request changing items. Is this here really wanted?

Reply:
I merged it from the previous branch with the new sorted items, so it does not really matter i guess